### PR TITLE
microprofile-fault-tolerance-19 MP config API coordinates are wrong

### LINKED ISSUE #19: MP config API coordinates are wrong
Group id and Artifact id are not the lat one for config API.

### PR DESCRIPTION
**Pull request by antoinesd**
fixes #19

### CODE REVIEW

**Comment by Emily-Jiang** [comment]
@antoinesd please fix the IP validation error

**Comment by antoinesd** [comment]
done

File: tck/pom.xml [hunk 1]
```diff
@@ -48,12 +48,12 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.eclipse.microprofile.apis</groupId>
-            <artifactId>microprofile-config_1.0_api</artifactId>
+            <groupId>org.eclipse.microprofile.config</groupId>
```
Comment (Emily-Jiang) [inline tck/pom.xml]: <groupId>org.eclipse.microprofile.fault.tolerance</groupId>
    <artifactId>microprofile-fault-tolerance-api</artifactId>

Can you change to the above as I meant to depend on fault tolerance api not config api at the moment?

**Comment by Emily-Jiang** [comment]
Thanks @antoinesd ! I'll work on the Tcks soon. Look forward to seeing more contributions from you!

**Pull request merged**